spidermon/contrib/scrapy/pipelines.py

```diff
@@ -31,7 +40,7 @@ def __init__(
         add_errors_to_items=DEFAULT_ADD_ERRORS_TO_ITEM,
         errors_field=None,
     ):
-        self.drop_items_with_errors = drop_items_with_errors
+        self.drop_items_with_errors = drop_items_with_errors or DEFAULT_DROP_ITEMS_WITH_ERRORS
```

Comment:
Why this change?
The default value of `drop_items_with_errors` is already `DEFAULT_DROP_ITEMS_WITH_ERRORS`

Reply:
I thought as add_items_with_errors is also structured like this ... Quite honestly, didn't think about this change. Will change both of them.

Reply:
Changed both, resolving. Let me know otherwise if there is any other change. 
Resolving.